### DISCUSSION TITLE
intelmq.bots.parsers.generic.parser_csv for only one column without title

### DISCUSSION THREAD

**Discussion by vandaref** [certtools/intelmq#2464] [Q&A]
Hi there,
I collect data from : http://rules.emergingthreats.net/blockrules/compromised-ips.txt
And I try to parse it with **intelmq.bots.parsers.generic.parser_csv**. However the **.txt** dosen't have **column title** and there is only **one column**. 
Do you have any idea how to setup my parser bot in runtime.yaml ?
Thanks team

**Comment by sebix** [comment]
Set
- `columns` to `source.ip`
- `skip_header` to false
